hashtables/ex1/ex1.py

```diff
@@ -13,6 +13,18 @@ def get_indices_of_item_weights(weights, length, limit):
     YOUR CODE HERE
     """
```

Comment:
You were pretty close on this just had some extra code so good work. Even when your wrong your always pretty close to the right answer.